resources/core/charts/api-controller/templates/tests/test-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.tests.enabled }}
```

Comment:
You shouldn't add it here. You should create separate PR which changes in charts. This PR must also contain also bump of tests image. Maybe it is not critical in this particular case as creating new namespace wouldn't break tests, but for the sake of keeping up to the process I would move it to the second PR with image change.

I wish it was simpler :D